prepare-cluster/action.yml

```diff
@@ -21,25 +24,41 @@ inputs:
   kubernetes-version:
     description: 'Kubernetes version to provision (format is distribution dependent).'
     required: false
+  cluster-name:
+    description: 'Name of the cluster to provision'
+    required: false
+    default: ''
   ttl:
     description: 'Cluster TTL (duration, max 48h)'
     required: false
-  helm-values:
-    description: 'A Helm values.yaml file to use'
+  disk:
+    description: 'Disk size in GiB'
     required: false
-  helm-chart-name:
-    description: 'The name of the Helm chart to use'
+  nodes:
+    description: 'Number of nodes to provision'
+    required: false
+  instance-type:
+    description: 'Instance type to provision'
+    required: false
+  timeout-minutes:
+    description: 'Time to wait for the cluster to have a status of `running`'
     required: false
-  helm-run-preflights:
-    description: 'Run preflight checks (true/false)'
+    default: '20'
+  tags:
+    description: |
+      Tags to assign to the cluster.
+      Example:
+      <pre>tags: \|
+        - key: "department"
+          value: "engineering"</pre>
     required: false
-    default: 'true'
-  kots-config-values:
-    description: 'The KOTS config values to use'
+  kubeconfig-path:
+    description: 'If specified, the kubeconfig will be written to this path'
     required: false
-  kots-wait-duration:
-    description: 'Timeout for KOTS to be used while waiting for individual components to be ready. must be in Go duration format (eg: 10s, 2m) (default "2m")'
+  export-kubeconfig:
+    description: 'Export the KUBECONFIG variable (true/false)'
     required: false
+    default: 'false'
```

Comment:
Shouldn't you also use these in the code of the action?

Reply:
there is no code is there?